test/oban/plugins/cron_test.exs

```diff
@@ -16,6 +19,28 @@ defmodule Oban.Plugins.CronTest do
       assert_valid(crontab: [{"* * * * *", Worker, queue: "special"}])
     end
 
+    test "workers exitence is validated" do
+      assert_raise ArgumentError, "Fake worker is not found or can't be loaded", fn ->
+        Cron.validate!(crontab: [{"* * * * *", Worker}, {"* * * * *", Fake}])
+      end
+    end
+
+    test "workers perform/1 callback is validated" do
+      assert_raise ArgumentError,
+                   "Oban.Plugins.CronTest.WorkerWithoutPerform worker does not implement `perform/1` callback",
+                   fn ->
+                     Cron.validate!(crontab: [{"* * * * *", WorkerWithoutPerform}])
+                   end
```

Comment:
The more specific errors are great and with the suggested error wording these _should_ fit on fewer lines.